lib/peep.rb

```diff
@@ -0,0 +1,56 @@
+require 'pg'
+
+class Peep
+  attr_reader :id, :username, :message, :datetime
+
+  def initialize(id:, username:, message:, datetime:)
+    @id = id
+    @username = username
```

Comment:
Think that this is a great addition to the bot - would be more reflective of Twitter